tests/TestDepBuildsInSource/__snapshots__/test.js.snap

```diff
@@ -56,7 +54,6 @@ export dep__etc=\"$dep__install/etc\"
 # [BuiltIns]
 export testdepoutofsourcebuildisfalse__name=\"TestDepOutOfSourceBuildIsFalse\"
 export testdepoutofsourcebuildisfalse__version=\"1.0.0\"
-export testdepoutofsourcebuildisfalse__source_root=\"$esy__sandbox\"
```

Comment:
See https://github.com/jordwalke/esy/pull/24/files#diff-57f4be5f4025a4fd81a3bbc2cdfc2194L192

We discussed in Discord that `$*__source_root` is an is an impl. detail so I moved it to `esyBuildEjectCommand.js` under `$esy_build__source_root` name.